File: src/main/kotlin/com/sg/launcher/repository/UserTokenRepository.java
package com.sg.launcher.repository;

import com.sg.launcher.repository.entity.UserTokenEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Set;

public interface UserTokenRepository extends JpaRepository<UserTokenEntity, Long> {
    Set<UserTokenEntity> findByUserId(Long userId );

    UserTokenEntity findByUserToken(String userToken );
}

File: src/main/kotlin/com/sg/launcher/repository/UserTokenRepository.kt
package com.sg.launcher.repository

import com.sg.launcher.repository.entity.UserTokenEntity
import org.springframework.data.jpa.repository.JpaRepository

interface UserTokenRepository : JpaRepository<UserTokenEntity?, Long?> {
    fun findByUserId(userId: Long?): Set<UserTokenEntity?>?
    fun findByUserToken(userToken: String?): UserTokenEntity?
}